kafka/channel/config/configmaps/kafka-channel.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 The Knative Authors
```

Comment:
I prefer you keep the name of this file as `kafka-config`

Reply:
This configuration is pretty much general, not specific to kafka channels (we were planning to use it in eventing-kafka-broker too)